actors/user/src/main/java/org/sunbird/user/util/MigrationUtils.java

```diff
@@ -61,16 +61,28 @@ public static void markUserAsRejected(ShadowUser shadowUser) {
         updateRecord(propertiesMap, shadowUser.getChannel(), shadowUser.getUserExtId());
         ProjectLogger.log("MigrationUtils:markUserAsRejected:update in cassandra  with userExtId" + shadowUser.getUserExtId(),LoggerEnum.INFO.name());
     }
+    /**
+     * this method will mark the user Failed(3) in shadow_user table
+     * if the user doesn't want to migrate
+     * @param shadowUser
+     */
+    public static void updateClaimStatus(ShadowUser shadowUser,int claimStatus) {
```

Comment:
avoid using void method , that will be hard for unit test.

Reply:
done